packages/generate/src/edgeql-js/generateInterfaces.ts

```diff
@@ -178,12 +176,8 @@ export const generateInterfaces = (params: GenerateInterfacesParams) => {
   const plainTypesExportBuf = new CodeBuffer();
 
   const writeModuleExports = (module: ModuleData) => {
-    const wrapInNamespace = !(module.isRoot && module.name === "default");
-    if (wrapInNamespace) {
-      plainTypesCode.writeln([t`export namespace ${module.internalName} {`]);
-      plainTypesCode.writeln([js`const ${module.internalName} = {`]);
```

Comment:
Why are we dropping the `js` writing here? Looking at the changes here seems like it was only ever writing an empty object with the module name so seems safe, just curious really.

Reply:
I was just removing it as it seems to be unused, since the interfaces file only contains types, and there's no runtime code. And it looks like the ouput mode is hardcoded to only ever output the `ts` tagged code anyway: https://github.com/edgedb/edgedb-js/blob/master/packages/generate/src/interfaces.ts#L58-L62.